share/spice/spotify/spotify.js

```diff
@@ -0,0 +1,38 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_spotify = function(api_result){
+
+        var query = DDG.get_query();
+        query = query.replace(/spotify/, ""); //replace trigger words from query
+
+        // Validate the response (customize for your Spice)
+        if (!api_result || api_result.tracks.items.length === 0) {
```

Comment:
@JMPerez is it possible for `tracks` and `tracks.items` to not exist in the result? If that's possible, we might have to do something like

```javascript
if(!api_result || !api_result.tracks || ...
```